edenai_apis/apis/google/google_image_api.py

```
@@ -347,6 +347,148 @@ def image__face_detection(
             standardized_response=FaceDetectionDataClass(items=result),
         )
 
+    async def image__aface_detection(
+        self, file: str, file_url: str = "", **kwargs
```

Ruff (0.14.4)

Unused method argument: `file_url`

(ARG002)